ansible/roles/flannel/handlers/main.yml

```diff
@@ -5,7 +5,6 @@
     - stop docker
     - delete docker0
     - start docker
-  when: inventory_hostname in groups['nodes']
```

Comment:
Understood. As it stands, the docker bridge on the master is on the wrong subnet, but I suppose as long as we don't actually use docker there, it can last until the next reboot.